tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/RootCommands.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+using System;
+using System.Threading.Tasks;
+using NewRelic.Agent.IntegrationTests.Shared.ReflectionHelpers;
+
+namespace MultiFunctionApplicationHelpers
+{
+    [Library]
+    public static class RootCommands
+    {
+        [LibraryMethod]
+        public static void DelaySeconds(int seconds)
```

Comment:
Nice! Surprised we didn't have this

Reply:
Yeah, I could have sworn we did at one point.